recipe/meta.yaml

```diff
@@ -19,7 +19,8 @@ requirements:
     - pip
   run:
     - python
-    - iris >=2
+    - numpy >=0.15
+    - iris >=3.0.0rc0
```

Comment:
@pp-mo Shouldn't this just be completely opinionated i.e., `iris 3.0.0rc0`

Reply:
? I don't agree.
When Iris 3.0 is out, it should work fine with that.